core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSequencerReference.java

```diff
@@ -84,27 +114,12 @@ public Platform getPlatform() {
   }
 
   @Override
-  public void setAvailable(Boolean available) {
-    this.available = available;
-  }
-
-  @Override
-  public Boolean getAvailable() {
-    return this.available;
-  }
-
-  @Override
-  public void checkAvailability(int timeout) throws IOException {
-    this.available = getIpAddress() == null ? false : getIpAddress().isReachable(timeout);
-  }
-
-  @Override
-  public void setIpAddress(InetAddress ip) {
+  public void setIpAddress(String ip) {
```

Comment:
Maybe validate the IP address in the setter? Otherwise this can be anything and will need to be validated every time it's used, before getFQDN() is called, etc